.github/workflows/release.yml

```diff
@@ -9,6 +9,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: npm install
+      - run: npm version "$(basename ${{ github.ref }})"
```

Comment:
tagging not to be automated (i.e. `npm version M.m.p && git push && git push --tags` to be run locally) for now

```suggestion
```